test/e2e/queue.go

```diff
@@ -733,8 +733,8 @@ var _ = Describe("AppWrapper E2E Test", func() {
 
 		context := initTestContext()
 		var aws []*arbv1.AppWrapper
-		//appwrappersPtr := &aws
```

Comment:
To me there should be a clean up here. I don't know why it was removed in the first place, but to be consistent with the rest of the tests the lines
```
appwrappersPtr := &aws
defer cleanupTestObjectsPtr(context, appwrappersPtr)
```
should be there.

Reply:
the original code had it commented out, I did experiment with adding the cleanup, yet the tests would fail as mcad would die / panic miserably. I propose we address the underlying cause with a separate PR.

Reply:
Sounds good. This is just so weird

Reply:
Added issue #311